v2/routing_services.proto

```diff
@@ -0,0 +1,239 @@
+// Copyright © 2020 The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package org.packetbroker.v2;
+
+import "google/protobuf/empty.proto";
+import "packetbroker/api/v2/enums.proto";
+import "packetbroker/api/v2/messages.proto";
+import "packetbroker/api/v2/routing.proto";
+
+option go_package = "go.packetbroker.org/api/v2;packetbroker";
+
+message ListTenantsRequest {
+  // LoRa Alliance NetID of the Member.
+  uint32 net_id = 1;
+  // Number of items to skip for pagination.
+  uint32 offset = 2;
+  // Limit number items for pagination.
+  // If 0, use the server's default.
+  uint32 limit = 3;
+}
+
+message ListTenantsResponse {
+  repeated string tenant_ids = 1;
```

Comment:
I think it's better to use a `repeated Tenant` instead of only the IDs. Otherwise clients will very likely have to do multiple `TenantManager.GetTenant` requests after the List request.